tools/build_file_generator/build_file_generator.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package com.google.startupos.tools.buildfilegenerator;
+
+option java_package = "com.google.startupos.tools.buildfilegenerator";
+option java_outer_classname = "Protos";
+
+message Import {
+  string root_dir = 1;
+  repeated string subdir = 2;
+  string imported_class_name = 3;
+  bool whole_package_import = 4;
+  bool standard_java_package = 5;
+}
+
+message JavaClass {
+  repeated Import import = 1;
```

Comment:
We can get the class name from the filename, right?
We'll also need the package, which you can parse as well.